### DISCUSSION TITLE
EAD ou Presencial - Analise e desenvolvimento de sistemas

### DISCUSSION THREAD

**Discussion by cmanoel22** [frontendbr/forum#787] [Discussão]
Estou avaliando EAD de ADS, só que quanto mais pesquiso mais preocupado fico, as opções que meu  bolso alcança seria Cruzeiro do sul, Estácio, Unip, alguém estuda EAD em alguma delas pra recomendar ou falar "fuja" se não puder indicar uma EAD que seja razoável, presencial minhas opções por conta da localização e valor seria FMU e UNIP, nenhuma delas sobrevive ao reclame aqui, infelizmente não tenho grama pra bancar FIAP, Senac, Mackenzie...

**Comment by tarciozemel** [comment]
Se não estiver almejando alguma vaga específica que exija diplomação, esqueça faculdade.

Você vai aprender muito mais (investindo muito menos) através de cursos avulsos.

**Comment by n3pu** [comment]
Vou contar minha passagem acadêmica. Passei numa federal longe de onde morava e não tinha condições de morar lá. Então peguei uma bolsa integral numa particular. Presencial ADS na Estácio. Esse presencial contém cadeiras online. Essas cadeiras online não são muito boas, ja peguei conteúdo muito antigo, falando do tempo do msn. Fora que vc tem q ter foco. Deixar pra amanhã? Não rola. Matéria online se deixar pra depois acumula. Algumas provas são presenciais, pelo menos 1 prova presencial pra cada cadeira online. Fora isso, como não pago diretamente, ouço bastante gente reclamando dos boletos q vão errados ou com informações duvidosas.

Uma dica, tenho 26 anos e termino esse ano com exatamente 2,5 anos de curso (isso mesmo, o curso tem duração mínima de 2,5 anos). Se vc não focar, vai acabar atrasando o término. Agora, prefira a opção de bolsa, pq é bastante complicado ficar atrás de resolver pendência de pagamento.

Outra coisa. @tarciozemel falou em almejar vaga com esse requisito. Eu fiz curso por fora, passei 2 anos de técnico e 3 anos de design, mas vida acadêmica é uma coisa totalmente fora do que se imagina antes de entrar. Se você entrar, tenha ciência de q lá vc pode seguir por vários caminhos e vc tem q se focar em estudar. Vc vai ver coisas q não quer estudar, mas vai ter q estudar pra passar. Mas depois vai ver aquilo servindo pra outros propósitos. Vc vai ver q tem gente focando na vida acadêmica, de pequisa e artigos, mestres, doutores. O mais interessante disso é o **networking**. **Se apegue a pessoas de valores agregantes e descarte aqueles que nada agregam**. Participe e se envolva. Sim, mas e o dinheiro? Amigo, isso vem com o tempo, confie e foque! Daqui a pouco chega sua oportunidade se vc se tornar apto.

É fácil? Lógico q não. Se fosse fácil, todo mundo fazia isso. Mas vale muito a pena.

Faça a sua decisão e foque. Sucesso!

**Comment by tarciozemel** [comment]
Verdade, @ewerto0n, uma das únicas coisas que prestam em faculdade é o networking. Como exemplo, hoje em dia tenho um empresa em que um dos sócios é um colega da época da faculdade.

Mas já me formei há muito tempo. Atualmente, através de redes sociais, espaços de coworking etc., já é possível ter acesso a um networking potencial muito maior (e de mais qualidade) que em faculdades.

**Comment by kivervinicius** [comment]
@ewerto0n concordo com tudo, fora que o ambiente proporciona vivencias bem diferentes do "mundo real"

**Comment by klarkc** [comment]
Cara, fiz e se fosse hj n faria mais n, tá 10% eu aproveitei pra algo, n vou ser ingrato. Porém nessa carreira vale muito mais a pena participar de Meetups, Feiras, Conferencias etc para fazer networking do que fazer faculdade, ainda mais EAD, q vc tem pouquíssimo contato com colegas e professores. Mas se vc precisa de alguém que lhe ensine o caminho das pedras, comece, no momento em que achar q está lhe atrapalhando deixe-a sem pensar duas vezes. A não ser q vc pense em fazer concurso público ou ir para o exterior.

**Comment by woliveiras** [comment]
Acredito que networking nós fazemos com muito mais qualidade em meetups. Tenho grandes amizades que vem durando anos que fiz através de um meetups. :smile: 

Por isso acredito que a faculdade acabaria inútil se formos fazer uma graduação só para conhecer pessoas.

Eu reforço as palavras do @tarciozemel com a mesma opinião de que só vale a pena faculdade se for por que alguma vaga que você almeja obriga o certificado. 

Hoje eu faço EAD pela UNINTER. Não é uma das faculdades mais famosas do país, porém foi a que a grade mais me agradou (se é pra eu ficar 2,5 anos da minha vida fazendo um curso desses, que pelo menos tenha matérias que eu curto).

Os materiais parecem ser regravados anualmente, apesar da qualidade do vídeo não ser aquelas coisas. O professor de bancos de dados, mesmo, estava comentando em uma aula sobre NoSQL com MongoDB e falando sobre Big Data e os casos de empresas como Uber e Airbnb. O de matemática comentou sobre a, não tão recente mais, alteração do Whatssapp para criptografia ponta a ponta. Enfim, parece ser um material legal (não tenho muitos parâmetros pra comparar).

Porém todos os dias da minha vida eu penso em largar o curso, pois as vagas que eu conheço, em empresas legais de se trabalhar, não pedem diploma. :joy:

**Comment by lidimayra** [comment]
Sou formada em Análise e Desenvolvimento de Sistemas pela FATEC Jundiaí. Eu recomendo fortemente.

Acho importante pontuar o fato de que faculdade de tecnologia não se limita a técnicas de computação. Concordo que determinados tópicos podem ser alcançados através de cursos avulsos e de forma muito mais customizada aos interesses de cada um. Mas para a formação do profissional completo, vejo ensino superior como peça-chave.

Estimula senso crítico, aplicação de método científico, elaboração de pesquisa, raciocínio amplo, habilidades de gestão, trabalho em equipe... a participação em atividades opcionais atreladas ao curso também fornecem vivências extremamente valiosas.

**Comment by tcelestino** [comment]
Experiência própria: tranquei a faculdade há um bom tempo e pretendo voltar só para aumentar as oportunidades de trabalho (inclusive, fora do país). Fazia ADS em Salvador, tranquei por causa de um trabalho e nunca mais voltei. Já trabalhava na área há um certo tempo e realmente como o pessoal falou, a faculdade me acrescentou muita coisa teorica, mas do resto, vi nada além disso. Estudar sempre é bom, se quer ter o diploma, vá em busca. Eu não conseguiria estudar online, mas é questão de gosto. :)

Aproveitando: faculdade não quer dizer EMPREGO dos sonhos. Já vimos vários casos (nem precisa ser fora do país) que se você for um bom profissional, não é a faculdade que vai dizer alguma coisa, até porque, já sabemos que uma galera tem faculdade e não tem qualidade em muitos aspectos.
